tasks/main.yml

```diff
@@ -10,9 +18,16 @@
   when: ansible_os_family == "RedHat"
 
 - name: Configure plugins repository (apt)
-  apt_repository:
-    repo: "{{ foreman_scap_client_apt_repo_url }}"
-    state: "{{ foreman_scap_client_repo_state }}"
+  block:
+    - name: Configure gpg_key for client repository (apt)
```

Comment:
This works only, if apt-key (deprecated on new debian) or gpg is installed: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html#requirements

Unfortunately, debian 13 does not install gpg by default. I would recommend to add a package install section to install gpg for debian / ubuntu hosts. Can you adapt this @dgoetz ?